client/my-sites/site-settings/manage-connection/index.jsx

```diff
@@ -15,39 +14,21 @@ import DocumentHead from 'components/data/document-head';
 import HeaderCake from 'components/header-cake';
 import Main from 'components/main';
 import SiteOwnership from './site-ownership';
-import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
-import { isJetpackSite } from 'state/sites/selectors';
-import { isSiteAutomatedTransfer } from 'state/selectors';
+import redirectNonJetpack from 'my-sites/site-settings/redirect-to';
 
 class ManageConnection extends Component {
-	componentDidMount() {
-		this.verifySiteIsJetpack();
-	}
-
-	componentDidUpdate() {
-		this.verifySiteIsJetpack();
-	}
-
-	verifySiteIsJetpack() {
-		if ( this.props.siteIsJetpack === false || this.props.siteIsAtomic ) {
-			this.redirectToGeneral();
-		}
-	}
-
-	redirectToGeneral = () => {
-		const { siteSlug } = this.props;
-
-		page( '/settings/general/' + siteSlug );
-	};
 
```

Comment:
Nit: although our linter is not reporting it, this is an empty line in the beginning of a code block, so it should be removed.